### ISSUE TITLE
Step input increment/decrement gets disabled on value removal

### ISSUE THREAD

**Issue by pavan-kumar-deshkulkarni** [SAP/fundamental-ngx#8831]
#### Is this a bug, enhancement, or feature request?
Bug/ feature request

#### Briefly describe your proposal.
When you remove the value from step input -+ gets disabled. Ideally it should be enable and on click it should reset to 0. Disable give an impression that you can use that field.

https://user-images.githubusercontent.com/115621723/195296236-50c9c893-8df3-4847-9672-0f5aa485ac9a.mov
